<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addEvent</class>
 <widget class="QDialog" name="addEvent">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>295</width>
    <height>345</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>17</x>
     <y>14</y>
     <width>203</width>
     <height>48</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="EventNameLabel">
      <property name="text">
       <string>Event Name:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="eventName"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="eventTypeLabel">
      <property name="text">
       <string>Event Type:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="eventType"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>18</x>
     <y>310</y>
     <width>158</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <item>
     <widget class="QPushButton" name="saveEvent">
      <property name="text">
       <string>Save Event</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancel">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTextEdit" name="textEdit_details">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>193</y>
     <width>257</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>72</y>
     <width>147</width>
     <height>119</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="eventDateLabel">
      <property name="text">
       <string>Event Date:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QDateEdit" name="eventDate">
      <property name="dateTime">
       <datetime>
        <hour>0</hour>
        <minute>0</minute>
        <second>0</second>
        <year>2015</year>
        <month>11</month>
        <day>18</day>
       </datetime>
      </property>
      <property name="maximumDateTime">
       <datetime>
        <hour>23</hour>
        <minute>59</minute>
        <second>59</second>
        <year>7000</year>
        <month>12</month>
        <day>31</day>
       </datetime>
      </property>
      <property name="maximumTime">
       <time>
        <hour>23</hour>
        <minute>59</minute>
        <second>59</second>
       </time>
      </property>
      <property name="date">
       <date>
        <year>2015</year>
        <month>11</month>
        <day>18</day>
       </date>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="eventDateEndLabel">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>End Date:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QDateEdit" name="eventDateEnd">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="dateTime">
       <datetime>
        <hour>0</hour>
        <minute>0</minute>
        <second>0</second>
        <year>2015</year>
        <month>11</month>
        <day>18</day>
       </datetime>
      </property>
      <property name="maximumDateTime">
       <datetime>
        <hour>23</hour>
        <minute>59</minute>
        <second>59</second>
        <year>7000</year>
        <month>12</month>
        <day>31</day>
       </datetime>
      </property>
      <property name="maximumTime">
       <time>
        <hour>23</hour>
        <minute>59</minute>
        <second>59</second>
       </time>
      </property>
      <property name="date">
       <date>
        <year>2015</year>
        <month>11</month>
        <day>18</day>
       </date>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="startTimeLabel">
      <property name="text">
       <string>Start Time:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QTimeEdit" name="timeEdit_eventStart"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="endTimeLabel">
      <property name="text">
       <string>End Time:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QTimeEdit" name="timeEdit_eventEnd"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="eventEndLabel_2">
      <property name="text">
       <string>Details:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>174</x>
     <y>74</y>
     <width>114</width>
     <height>71</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QCheckBox" name="checkBox_multipleDay">
      <property name="text">
       <string>Multiple Day Event</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox_allDay">
      <property name="text">
       <string>All Day Event</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTextEdit" name="textEdit_details_2">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>179</y>
     <width>257</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>203</width>
     <height>48</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout_3">
    <item row="0" column="0">
     <widget class="QLabel" name="EventNameLabel_2">
      <property name="text">
       <string>Event Name:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="eventName_2"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="eventTypeLabel_2">
      <property name="text">
       <string>Event Type:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="eventType_2"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>157</x>
     <y>60</y>
     <width>114</width>
     <height>71</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QCheckBox" name="checkBox_multipleDay_2">
      <property name="text">
       <string>Multiple Day Event</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox_allDay_2">
      <property name="text">
       <string>All Day Event</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_4">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>58</y>
     <width>147</width>
     <height>119</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout_4">
    <item row="0" column="0">
     <widget class="QLabel" name="eventDateLabel_2">
      <property name="text">
       <string>Event Date:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QDateEdit" name="eventDate_2">
      <property name="dateTime">
       <datetime>
        <hour>0</hour>
        <minute>0</minute>
        <second>0</second>
        <year>2015</year>
        <month>11</month>
        <day>18</day>
       </datetime>
      </property>
      <property name="maximumDateTime">
       <datetime>
        <hour>23</hour>
        <minute>59</minute>
        <second>59</second>
        <year>7000</year>
        <month>12</month>
        <day>31</day>
       </datetime>
      </property>
      <property name="maximumTime">
       <time>
        <hour>23</hour>
        <minute>59</minute>
        <second>59</second>
       </time>
      </property>
      <property name="date">
       <date>
        <year>2015</year>
        <month>11</month>
        <day>18</day>
       </date>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="eventDateEndLabel_2">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>End Date:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QDateEdit" name="eventDateEnd_2">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="dateTime">
       <datetime>
        <hour>0</hour>
        <minute>0</minute>
        <second>0</second>
        <year>2015</year>
        <month>11</month>
        <day>18</day>
       </datetime>
      </property>
      <property name="maximumDateTime">
       <datetime>
        <hour>23</hour>
        <minute>59</minute>
        <second>59</second>
        <year>7000</year>
        <month>12</month>
        <day>31</day>
       </datetime>
      </property>
      <property name="maximumTime">
       <time>
        <hour>23</hour>
        <minute>59</minute>
        <second>59</second>
       </time>
      </property>
      <property name="date">
       <date>
        <year>2015</year>
        <month>11</month>
        <day>18</day>
       </date>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="startTimeLabel_2">
      <property name="text">
       <string>Start Time:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QTimeEdit" name="timeEdit_eventStart_2"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="endTimeLabel_2">
      <property name="text">
       <string>End Time:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QTimeEdit" name="timeEdit_eventEnd_2"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="eventEndLabel_3">
      <property name="text">
       <string>Details:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_5">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>296</y>
     <width>158</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_7">
    <item>
     <widget class="QPushButton" name="saveEvent_2">
      <property name="text">
       <string>Save Event</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancel_2">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>textEdit_details</zorder>
  <zorder>textEdit_details_2</zorder>
  <zorder>layoutWidget_2</zorder>
  <zorder>layoutWidget_3</zorder>
  <zorder>layoutWidget_4</zorder>
  <zorder>layoutWidget_5</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
